src/app/public/modules/status-indicator/status-indicator.component.html

```diff
@@ -0,0 +1,31 @@
+<span class="sky-status-indicator">
+  <span
+    aria-hidden="true"
```

Comment:
I'm not sure we want to do this - it would hide the text inside the `<ng-content>` from users with assistive tech. We actually have an issue filed here: https://github.com/blackbaud/skyux-theme/issues/133
According to the suggestion, we could consider some default text for each type, but allow consumers to override. What do you think?

Reply:
The `aria-hidden` attribute is limited to the icon itself, not the message displayed with `ng-content`, but that issue you linked does make a good point about how the severity should be conveyed to assistive technology. I'll go ahead and address that issue with this PR.